packages/corvid-local-server/test/it/without-corvid-types.spec.js

```diff
@@ -0,0 +1,48 @@
+const { editorSiteBuilder } = require("corvid-fake-local-mode-editor");
```

Comment:
maybe move these tests to the code-completion.spec.js file ?

Reply:
i had a technical problem to move those tests to the code-completion.spec.js file.i didn't succeed running all tests (with and without jest.mock) in the same file..